pkg/controller/sub_controller/be/controller.go

```diff
@@ -99,6 +100,11 @@ func (be *Controller) Sync(ctx context.Context, dcr *v1.DorisCluster) error {
 		return err
 	}
 
+	// check if the secret exists in the namespace
+	if dcr.Spec.BeSpec.Secrets != nil {
```

Comment:
this check should before "applystatefulset" and should output an event on dorislcuster.

Reply:
Done